app/assets/stylesheets/grader.scss

```diff
@@ -17,7 +17,7 @@
 }
 
 .annotation_text_display {
-  @include border-radius(5px);
+  border-radius: $radius;
```

Comment:
Properties should be ordered background, border, border-radius, max-width, padding, position, white-space, width, z-index